views/js/runner/plugins/security/preventScreenshot.js

```diff
@@ -115,16 +117,22 @@ define([
                 .on('keyup' + '.' + this.getName(), function (e) {
                     if (e.key === 'PrintScreen') {
                         triggerCopyEvent();
-
+                        var context = testRunner.getTestContext();
```

Comment:
Please declare variables at the top of their scope to reflect hoisting, ie.
```js
.on('keyup' + '.' + this.getName(), function (e) { 
    var context;
    if(...){
      context = ...
    }
}
```
(ESLint should have warned you about this)